auslib/admin/views/forms.py

```diff
@@ -111,16 +120,27 @@ def _validator(form, field):
         # empty input
         if field.data is None:
             return
-        try:
-            op, operand = get_op(field.data)
-            version = MozillaVersion(operand)
-        except ValueError:
-            raise ValidationError("ValueError. Couldn't parse version for %s. Invalid '%s' input value" % (field.name, field.name))
-        except:
-            raise ValidationError('Invalid input for %s . No Operator or Match found.' % field.name)
-        # MozillaVersion doesn't error on empty strings
-        if not hasattr(version, 'version'):
-            raise ValidationError("Couldn't parse the version for %s. No attribute 'version' was detected." % field.name)
+        rulesVersionList = field.data.split(",")
+        has_more_than_one_version = 1 < len(rulesVersionList)
```

Comment:
This block looks like it will work, but I think it would be more readable if you returned early if there's only one version, and only used the loop if there's more than one.

Some tests in https://github.com/mozilla/balrog/blob/master/auslib/test/admin/views/test_rules.py are necessary too. Even if it's perfect now, we need to make sure we don't accidentally break it later!